modules/client-configuration/pages/clients-rh-rhui.adoc

```diff
@@ -0,0 +1,424 @@
+----
+
+
+
+== Check Synchronization Status
+
+ifeval::[{suma-content} == true]
+
+
+include::snippets/check_sync_webui_suma.adoc[]
+
+
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+
+
+include::snippets/check_sync_webui_uyuni.adoc[]
+
+
+endif::[]
+
+
+include::snippets/check_sync_cli.adoc[]
+
+
+[NOTE]
+====
+{rhel} channels can be very large.
+Synchronization can sometimes take several hours.
+====
+
+
+.Procedure: OPTIONAL: Creating a Salt State to Deploy Configuration Files
+
+. On the {productname} Server {webui}, navigate to menu:Configuration[Channels].
+. Click btn:[Create State Channel].
+* In the [guimenu]``Name`` field, type [systemitem]``subscription-manager: disable yum plugins``.
+* In the [guimenu]``Label`` field, type [systemitem]``subscription-manager-disable-yum-plugins``.
+* In the [guimenu]``Description`` field, type [systemitem]``subscription-manager: disable yum plugins``.
+* In the [guimenu]``SLS Contents`` field, leave it empty.
+. Click btn:[Create Config Channel]
+. Click btn:[Create Configuration File]
+* In the [guimenu]``Filename/Path`` field type [systemitem]``/etc/yum/pluginconf.d/subscription-manager.conf``.
+* In the [guimenu]``File Contents`` field type:
+----
+[main]
+enabled=0
+----
+. Click btn:[Create Configuration File]
+. Take note of the value of the field [guimenu]``Salt Filesystem Path```.
+. Click on the name of the Configuration Channel.
+. Click on [guimenu]``View/Edit 'init.sls' File``
+* In the [guimenu]``File Contents`` field, type:
+----
+configure_subscription-manager-disable-yum-plugins:
+  cmd.run:
+    - name: subscription-manager config --rhsm.auto_enable_yum_plugins=0
+    - watch:
+      - file: /etc/yum/pluginconf.d/subscription-manager.conf
+  file.managed:
+    - name: /etc/yum/pluginconf.d/subscription-manager.conf
+    - source: salt:///etc/yum/pluginconf.d/subscription-manager.conf
+----
+. Click btn:[Update Configuration File].
+
+
+[NOTE]
+====
+The ``Creating a Salt State to Deploy Configuration Files`` procedure is optional.
+====
+
+
+.Procedure: Creating a System Group for {rhel} Clients
+
+. On the {productname} Server {webui}, navigate to menu:Systems[System Groups].
+. Click btn:[Create Group].
+* In the [guimenu]``Name`` field, type [systemitem]``rhel-systems``.
+* In the [guimenu]``Description`` field, type [systemitem]``All RHEL systems``.
+. Click btn:[Create Group].
+. Click [guimenu]``States`` tab.
+. Click [guimenu]``Configuration Channels`` tab.
+. Type [systemitem]``subscription-manager: disable yum plugins`` at the search box.
+. Click btn:[Search] and the state will appear.
+. Click the checkbox for the state at the [systemitem]``Assign`` column.
+. Click btn:[Save changes].
+. Click btn:[Confirm].
+
+If you already have RHEL systems added to {productname}, assign them to the new system group, and then apply the highstate.
+
+
+
+.Procedure: Adding the System Group to Activation Keys
```

Comment:
Same remark again.

Reply:
This one does not make sense on the RHUI context.